core-java-modules/core-java-concurrency-simple/src/main/java/com/baeldung/concurrent/executorservice/ExecutorServiceDemo.java

```diff
@@ -24,4 +24,9 @@ public void execute() {
 
     }
 
+    public static void main(String... args) {
+        ExecutorServiceDemo demo = new ExecutorServiceDemo();
```

Comment:
where is this code snippet used in the article?

Reply:
Removed